book/garbage-collection.md

```diff
@@ -392,8 +392,8 @@ We implement that in the "table" module:
 
 ^code mark-table
 
-Pretty straightforward. We walk the entry array. For each one, we its value. We
-also mark the key strings for each entry since the GC manages those strings too.
+Pretty straightforward. We walk the entire array. For each one, we mark its value.
```

Comment:
I'm not sure the first one here was a typo. It's the array of entries after all.

Reply:
Think you're right; I'll put that back.

Reply:
Thin you're right; I put it back.